bin/pyenv-installer

```diff
@@ -22,7 +22,7 @@ colorize() {
 }
 
 checkout() {
-  [ -d "$2" ] || git clone "$1" "$2"
+  [ -d "$2" ] || git clone --depth 1 "$1" "$2"
```

Comment:
I don't like to use `--depth 1` by default. Though it might be useful if it is configurable for some cases. Please use `PYENV_GIT_CLONE_OPTIONS` here. It should allows users to configure it on runtime.

```sh
$ curl -L https://raw.githubusercontent.com/yyuu/pyenv-installer/master/bin/pyenv-installer | PYENV_GIT_CLONE_OPTIONS="--depth 1" bash
```